README.md

```diff
@@ -43,7 +43,7 @@ require('apostrophe')({
 ### Pages
 ![Screenshot highlighting the export menu item in the context menu of a page in the manager.](https://static.apostrophecms.com/apostrophecms/import-export/images/page-export.png)
 
-Pages can currently only be exported singly. Support for exporting multiple pages at once is planned for a future release. When exporting a page it will retain its publication status upon import. A page that is published when exported will be published on import, while draft pages will remain in draft status. In order to export a page, you use the context menu to the right of each page in the manager.
+Pages can currently only be exported singly. Support for exporting multiple pages at once is planned for a future release. When exporting a page it will retain its publication status upon import. A page that is published when exported will be published on import unless you select the option to only import as a draft document. Draft pages will remain in draft status. In order to export a page, you use the context menu to the right of each page in the manager.
```

Comment:
Batch uploading is supported for pages too now.

<img width="759" alt="image" src="https://github.com/user-attachments/assets/5895568d-7838-40f4-bcca-e6c50cf80177" />

Reply:
Updated